{"author":{"id":"faefbc3bb716da5014722ece9ba931cfbf9f50bc38d7d5d9529b66d426296fee"},"ops":[{"type":1,"timestamp":1571258987,"nonce":"uh1T5uzJ72zjmvKc9V8LFr4tcWY=","metadata":{"github-id":"MDU6SXNzdWU1MDgwOTE2OTE=","github-url":"https://github.com/git-bug/git-bug/issues/230","origin":"github"},"title":"import error: issue creation: Multiple matching bug found","message":"I cloned the git-bug repo, ran `git bug pull`, then configured the bridge and ran `git bridge pull` and got this error:\n\n```\n‚ùØ gu bridge pull\nimport error: issue creation: Multiple matching bug found:\n4261bebb63ffaac33416003a2aa62a5a12df01b0\n678b0eccf6bae77dc47ec71804b6a58f3cfdc4aa\nSuccessfully imported 0 issues and 0 identities with default bridge\n```\n\nIs there a way for me to manually fix this? Should I have configured and pulled the bridge first then run `git bug pull`?","files":null}]}